.github/ISSUE_TEMPLATE/kubernetes_bump.md

```diff
@@ -26,24 +26,33 @@ changes should be cherry-picked to all release series that will support the new
     * `test/*`: search for occurrences of the previous Kubernetes version
     * `Tiltfile`
   * Ensure the latest available kind version is used (including the latest images for this kind release)
+    * Add new images in the [kind mapper.go](https://github.com/kubernetes-sigs/cluster-api/blob/48ae58e51f9723ab7b9635d0e05ee54c4843707a/test/infrastructure/kind/mapper.go#L79).
+      * See the [kind releases page](https://github.com/kubernetes-sigs/kind/releases) for the list of released images.
+    * Set new default image for the [test framework](https://github.com/kubernetes-sigs/cluster-api/blob/48ae58e51f9723ab7b9635d0e05ee54c4843707a/test/framework/bootstrap/kind_provider.go#L40)
   * Verify the quickstart manually
-  * Prior art: #7156
-  * bump `InitWithKubernetesVersion` and `WorkloadKubernetesVersion` in `clusterctl_upgrade_test.go`
+  * Bump `InitWithKubernetesVersion` and `WorkloadKubernetesVersion` in `clusterctl_upgrade_test.go`
+    * Note: Do not bump for upgrades using older clusterctl versions.
+  * Prior art: #9160
 * [ ] Ensure the jobs are adjusted to provide test coverage according to our [support policy](https://cluster-api.sigs.k8s.io/reference/versions.html#supported-kubernetes-versions):
-  * For the main branch and the release branch of the latest supported Cluster API minor release:
-    * Add new periodic upgrade job.
-    * Adjust presubmit jobs so that we have the latest upgrade jobs available on PRs.
   * For the main branch:
-    * periodics & presubmits: 
-      * Bump `KUBEBUILDER_ENVTEST_KUBERNETES_VERSION` of the `test-mink8s` jobs to the new minimum supported management cluster version.
     * periodics:
       * Bump `KUBERNETES_VERSION_MANAGEMENT` of the `e2e-mink8s` job to the new minimum supported management cluster version.
```

Comment:
I think this is also for the presubmit

Reply:
Great point. I only did it for the periodics 😭